src/lib/slider/_slider-theme.scss

```diff
@@ -5,7 +5,6 @@
 @mixin md-slider-theme($theme) {
   $accent: map-get($theme, accent);
 
-  // TODO(iveysaur): Find an implementation to hide the track under a disabled thumb.
```

Comment:
This `TODO` was actually referring to something else. If you look at the spec, you'll see that there's a tiny empty space around the thumb that covers up the track when the slider is disabled; this is talking about that.